nativelib/src/main/resources/scala-native/stackOverflowGuards.c

```diff
@@ -78,26 +80,26 @@ static struct sigaction *resolvePreviousSignalHandler(int sig) {
         return &previousSignalHandlers[1];
     default:
         fprintf(stderr,
-                "ScalaNative :: StackOverflowHandler does not define handler "
+                "%s StackOverflowHandler does not define handler "
                 "for %d signal\n",
-                sig);
+                snErrorPrefix, sig);
         exit(sig);
     }
 }
 
 static void protectStackGuardPage() {
     if (mprotect(currentThreadInfo.stackGuardPage, resolvePageSize(),
                  PROT_NONE) == -1) {
-        perror("ScalaNative Fatal Error :: StackOverflowHandler guard "
-               "protection failed");
+        perror(SN_FATAL_ERROR_MSG("StackOverflowHandler guard "
```

Comment:
If the goal is executable size, these are expanded by the preprocessor so each one will be laid down inline. It does the reuse with no computational overhead with no space saving. The only other alternative is to use `strcat` and handle the memory hit but since this for an error and exit? then that might be the best bet.

Reply:
To be honest, I'm not sure what's the best way to deal with this.

I started this PR for the space savings, although those are pretty minimal, and I do have mixed feelings about adding some manual memory management on the error handling :/

So I decided to just take the small win here of making error messages consistent... I still like the `constants.c` to have some savings where it's easy and to be able to pass the prefixes to the Scala side.